onchain/rollups/test/foundry/inputs/InputBox.t.sol

```diff
@@ -254,3 +271,22 @@ contract InputBoxTest is Test {
         assertEq(sum, totalNumOfInputs, "total number of inputs");
     }
 }
+
+contract LibInputCaller {
```

Comment:
You don't have to create a new contract for this. You can define in the test contract itself and call with an extra `this.` before, like here:

- Call: https://github.com/cartesi/rollups-contracts/blob/a61d5f736b208ad635a9b9ecf7ea086621a31ae8/onchain/rollups/test/foundry/dapp/CartesiDApp.t.sol#L723
- Definition: https://github.com/cartesi/rollups-contracts/blob/a61d5f736b208ad635a9b9ecf7ea086621a31ae8/onchain/rollups/test/foundry/dapp/CartesiDApp.t.sol#L705-L707